cli/ttparsers/ttcsv.go

```diff
@@ -0,0 +1,482 @@
+// Copyright 2011 The Go Authors. All rights reserved.
```

Comment:
Move the csv parser addition to a separate commit.

Reply:
Done